source/portal/v11Client.js

```diff
@@ -6,6 +6,7 @@
 
 var log = require('./logger').logger.getLogger('V11Client');
```

Comment:
For v1.2 spec, you should create v12Client.js for v12 specific logic.

Reply:
Also update protocol version for v12 client according to spec.

Reply:
A new file `client.js` is created in another PR.